### PR TITLE
enhancement: make verbose logging configurable

### PR DESCRIPTION
**Pull request by ra0x3**
### Description
- PR adds `-v --verbose` flag to `fuel-indexer` and `fuel-indexer-api-server`
- Updates docs accordingly

### Testing steps
- Ensure `forc-index start` points to a latest `fuel-indexer` binary
- Run with verbose logging
  - [ ] Try running the block explorer with and without the `-v --verbose` flag

 
```bash
cargo run --bin fuel-indexer -- run \
  --fuel-node-host beta-3.fuel.network \
  --fuel-node-port 80 \
  --manifest examples/block-explorer/explorer-indexer/explorer_indexer.manifest.yaml \
  --verbose
```
   - [ ] And the plugin as well (with and without verbosity)

```
cargo run --bin forc-index -- start -v
```

### Changelog
- enhancement: make verbose logging configurable

### CODE REVIEW

File: packages/fuel-indexer-lib/src/utils.rs
Comment (deekerno) [inline packages/fuel-indexer-lib/src/utils.rs:338]: `bool` implements the `Display` trait, so you should just be able to call `boolean_value.to_string()` instead of having to create a function. Am I missing something here?

File: plugins/forc-index/src/ops/forc_index_start.rs
Comment (deekerno) [inline plugins/forc-index/src/ops/forc_index_start.rs:64]: This needs to be changed before merging.

File: plugins/forc-index/src/ops/forc_index_start.rs
Comment (deekerno) [inline plugins/forc-index/src/ops/forc_index_start.rs:140]: Was this supposed to be changed from `anyhow::bail!` to `panic!`? If so, why?

**Review by deekerno — CHANGES_REQUESTED** [review]
Testing steps work. So much less spam now, thank you so much! 😭 

In addition to the review comments, it looks like `sqlx` is ignoring the `verbose` parameter of the database connection string.
```ignoring unrecognized connect parameter: verbose=true```
```ignoring unrecognized connect parameter: verbose=false```

It doesn't affect the functionality at all, but I just thought you should be aware.

File: packages/fuel-indexer-lib/src/utils.rs
Comment (ra0x3) [reply packages/fuel-indexer-lib/src/utils.rs:338]: 🤦🏽 brain fart

File: plugins/forc-index/src/ops/forc_index_start.rs
Comment (ra0x3) [reply plugins/forc-index/src/ops/forc_index_start.rs:140]: @deekerno 
- Yea. If you run `forc index start` when `fuel-indexer` is not in the path it fails, but the error gets swallowed  by `anyhow::bail` (the process will die and you'll get nothing in stdout)
- I think we can`panic` here (probably elsewhere) where we don't need to do anything after the error (though I definitely should include the error in the message)

**Comment by ra0x3** [comment]
> Testing steps work. So much less spam now, thank you so much! 😭
> 
> In addition to the review comments, it looks like `sqlx` is ignoring the `verbose` parameter of the database connection string. `ignoring unrecognized connect parameter: verbose=true` `ignoring unrecognized connect parameter: verbose=false`
> 
> It doesn't affect the functionality at all, but I just thought you should be aware.

@deekerno 
- Yea putting the param in the database URI is the only way I think to sneak `verbose_logging` into `IndexerConnection` w/o requiring a decent amount of api changes
- The warnings is obviously legit (since `verbose_logging` isn't an actual PostgreSQL URI param)
  - I didn't see anything [in the native list of URI params](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS) that would be a good substitute
- Definitely down to change it if there's a better method

**Comment by deekerno** [comment]
@ra0x3: If anything, you could probably use [`query_pairs_mut`](https://docs.rs/url/latest/url/struct.Url.html#method.query_pairs_mut) after getting the `verbose` value by creating an iterator from the original query pairs and then setting them all back through a map while filtering out `verbose`...but that's a bit of work for something that doesn't affect the functionality anyways. 😅

But everything looks good, I'll approve when tests pass.

**Comment by ra0x3** [comment]
> @ra0x3: If anything, you could probably use [`query_pairs_mut`](https://docs.rs/url/latest/url/struct.Url.html#method.query_pairs_mut) after getting the `verbose` value by creating an iterator from the original query pairs, clearing them on the original `url` object, setting them all back by iterating through the query iterator while filtering out `verbose` _and then_ use `url.to_string()`...but that's a bit of work for something that doesn't affect the functionality anyways. 😅
> 
> But everything looks good, I'll approve when tests pass.

@deekerno Ahh ok I see what you're saying - just grab the param w/o passing it. That's definitely possible. Change incoming

**Review by deekerno — APPROVED** [review]
:shipit:

**Pull request merged**
